aiida_zeopp/data/parameters.py

```diff
@@ -14,6 +14,7 @@
     'res': (bool, 'free_sphere_res'),
     'zvis': (bool, 'network_zvis'),
     'axs': (float, 'nodes_axs'),
+    'visVoro': (float, 'all_nodes_visvoro','acc_nodes_visvoro','nonacc_nodes_visvoro'),
```

Comment:
How about replacing `'label'` with `['label']` for all of those here?
Then you won't need to make an exception for visvoro anymore.

Reply:
Do you mean to change labels for all options to list, right?

Reply:
yes